plugins/xcode/xcode

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+export DOTFILES_XCODE_DEPS=('homebrew_core')
+
+dotfiles_xcode_apply () {
+  if ! mas list | grep -iq xcode; then
+    if ! command -v mas >/dev/null 2>&1; then
+      brew install mas
+    fi
+
+    log 'Installing Xcode. Note: this could take quite a while.'
+    # This is the app store id for xcode
+    mas install 497799835
```

Comment:
Not sure I follow having both`plugins/xcode/Brewfile` and manually installing it here?

Reply:
the homebrew plugin runs `brew cleanup` so it'll remove anything not in a Brewfile but we need `mas` and `xcode` installed before we run `brew bundle` because xcode invalidates any previously agreed to xcode agreements (and thus, `brew bundle` fails to install everything in the Brewfile after xcode).